shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/utils/LabelsBuilder.java

```diff
@@ -0,0 +1,147 @@
+package com.redhat.service.smartevents.shard.operator.v2.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.fabric8.kubernetes.client.utils.KubernetesResourceUtil;
+
+public class LabelsBuilder {
```

Comment:
FWIW I'd be tempted to move the existing `LabelsBuilder` from `-core` to `-v1` if it cannot be re-used/extended for `-v2`.

Reply:
Yep, good idea!

Reply:
Actually @manstis I've changed my mind for this PR. To move `LabelsBuilder` I also need to move `EventSourceFactory` but I'm likely to do a bit of refactoring on that in my next PR to make it usable across both versions of the Operator. So I'll leave things as they are for now and move it in my next PR.